pcapfile/savefile.py

```diff
@@ -102,14 +102,14 @@ def _load_savefile_header(file_h):
         byte_order = b'little'
         unpacked = struct.unpack('<IhhIIII', raw_savefile_header)
     else:
-        raise Exception('Invalid pcap file.')
+        raise UnknownMagicNumber("No supported Magic Number found")
```

Comment:
Sorry for the confusion, I meant in the message (→"No supported magic number found")